techmatters/flex-plugins
# Added a placeholder 'could not load' page if contact fails to create

## Description

Following on from yesterday's discussion I reworked the error handling case around not being able to create a contact

Now if a contact is not created when starting a task for any reason, a placeholder UI is put in place of the forms with 2 buttons, one to retry creating the contact, and another to complete the task

### Checklist
- N/A Feature flags added

### Related Issues
CHI-2320

### Verification steps

To test this you need to disable contact creation in HRM somehow, possibly just override the HRM URL in flex with an invalid oner
You should then check that the UI appears when acceopting a new task and the buttons work as expevcted
